### PR TITLE
bump manifest to 4.4

### LINKED ISSUE #311: Please prepare bundle for OCP 4.4
Now that we are fully branched for 4.4, please prepare your operator to supply a 4.4 bundle, so that 4.4 operator publishing works and doesn't overwrite 4.3 bundles. This means at least updating the package.yaml under https://github.com/openshift/cluster-logging-operator/tree/master/manifests

### PR DESCRIPTION
**Pull request by jcantrill**
bump the manifest to 4.4.  fixes #311

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @sosiouxme

File: manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.2.0 <4.3.0"
+    olm.skipRange: ">=4.2.0 <4.4.0"
```
Comment (sosiouxme) [inline manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml]: do you maybe want?
```suggestion
    olm.skipRange: ">=4.3.0 <4.4.0"
```

**Comment by sosiouxme** [comment]
but
/lgtm
/retest

**Pull request merged**